en/config_fw/weight_and_altitude_tuning.md

```diff
@@ -1,52 +1,63 @@
+## Vehicle Weight Compensation
+
+Set (both) the following parameters to scale the maximum climb rate, minimum sink rate, and adjust airspeed limits for weight:
 
-## Specify the Weight of the Vehicle
+- [WEIGHT_BASE](../advanced_config/parameter_reference.md#WEIGHT_BASE) — the weight of the vehicle at which the [Basic TECS tuning](../config_fw/position_tuning_guide_fixedwing.md#tecs-tuning-altitude-and-airspeed) was performed.
 
-[WEIGHT_BASE](../advanced_config/parameter_reference.md#WEIGHT_BASE) specifies the base weight of the vehicle at which the [TECS tuning](position_tuning_guide_fixedwing.md#tecs-tuning-altitude-and-airspeed) was performed.
-The weight of the vehicle in the tuning configuration should be measured with a scale and the parameter should be set accordingly.
+- [WEIGHT_GROSS](../advanced_config/parameter_reference.md#WEIGHT_BASE) — the actual weight of the vehicle at any given time, for example when using a larger battery, or with a payload that was not present during tuning.
 
-[WEIGHT_GROSS](../advanced_config/parameter_reference.md#WEIGHT_BASE) specifies the actual weight of the vehicle at any given time.
-This parameter can be set to reflect, for example, a payload attached to the vehicle that was not present during the tuning phase.
+You can determine the values by measuring the weight of the vehicle using a scale in the tuning configuration and when flying with a payload.
 
-When both [WEIGHT_BASE](../advanced_config/parameter_reference.md#WEIGHT_BASE) and [WEIGHT_GROSS](../advanced_config/parameter_reference.md#WEIGHT_GROSS) are set to a number larger than `0`, PX4 will use this information to scale the maximum climb rate, minimum sink rate, and adjust airspeed limits according to the weight ratio.
-The interested reader is referred to [the note section](../config_fw/weight_and_altitude_tuning.md#notes-to-the-derivation-of-weight-and-density-compensation) below for further details.
+Scaling is performed when _both_ `WEIGHT_BASE` and `WEIGHT_GROSS` are greater than `0`, and will have no effect if the values are the same.
+See the [algorithms](#weight-and-density-compensation-algorithms) section below for more information.
 
-## Compensation for Air Density
+## Air Density Compensation
 
 ### Specify a Service Ceiling
+
 In PX4 the service ceiling [FW_SERVICE_CEIL](../advanced_config/parameter_reference.md#FW_SERVICE_CEIL) specifies the altitude in standard atmospheric conditions at which the vehicle is still able to achieve a maximum climb rate of 0.5 m/s at maximum throttle and weight equal to [WEIGHT_BASE](../advanced_config/parameter_reference.md#WEIGHT_BASE).
 By default this parameter is disabled and no compensation will take place.
-This parameter needs to be estimated experimentally. It is always better to set a conservative value (lower value) than an optimistic value.
+
+This parameter needs to be determined experimentally.
+It is always better to set a conservative value (lower value) than an optimistic value.
```

Comment:
From "Effect of Density on Maximum Climb Rate"

> where  is the air density at sea level in the standard atmosphere and K is a scaling factor which determines the slope of the function. Rather than trying to identify this constants, the usual practice in aviation is to specify a service ceiling altitude at which the vehicle is still able to achieve a minimum specified climb rate

So my understanding is that while this scales by density ratio the scaling factor is hard to work out. So instead we set a service ceiling in which we know that the plane will fly OK. Is that right?

So what does the service ceiling do? I mean if I set this to 200m does this mean the vehicle is capped to that height?

Is there a rule of thumb about how you go about determining this? Can you suggest one? I.e. can you do some single maneuver and work it out from your logs?